prow/jobs/kyma/kyma-integration-gardener-aws.yaml

```diff
@@ -0,0 +1,39 @@
+# Kubectl 1.16
+base_image: &base_image
+  image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20200206-22eb97a4
+
+periodics:
+- name: kyma-integration-gardener-aws
+  decorate: true
+  decoration_config:
+    timeout: 14400000000000 # 4h
+    grace_period: 600000000000 # 10min
+  cron: "00 14 * * *"
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      path_alias: github.com/kyma-project/test-infra
+      base_ref: master
+  labels:
+    preset-gardener-aws-kyma-integration: "true"
+    preset-kyma-cli-stable: "true"
+    preset-kyma-keyring: "true"
```

Comment:
AFAIK the keyring and encryption key are used for let's encrypt certificates. Since we do not use them on Gardener, they should not be required here.

Reply:
Done thanks! Also removed it from the other gardener jobs.

Reply:
Great!